mlflow/server/js/src/experiment-tracking/components/App.css

```diff
@@ -92,6 +92,22 @@ div.docs {
   color: #ffffff;
 }
 
+.header-nav-link-experiments {
+  display: inline-block;
+  height: 39px;
```

Comment:
According to @gioa 's new design, we are currently keeping all px based measures a multiple of 4 when possible.